gene_benchmark/tasks.py

```diff
@@ -262,6 +281,10 @@ def run(self, error_score=np.nan):
 
         """
         outcomes, encodings = self._prepare_datamat_and_labels()
+        if is_binary_outcomes(self.task_definitions.outcomes):
+            outcomes = pd.get_dummies(self.task_definitions.outcomes).iloc[:, 0]
```

Comment:
Which class will be the zero and which will be the one? Is it alphabetical?

Reply:
yes. `get_dummies`  converts the classes by alphabetical order.